app/src/main/java/com/miniclip/footb/ui/model/ConfigData.kt

```diff
@@ -0,0 +1,8 @@
+package com.miniclip.footb.ui.model
+
+data class ConfigData(
+    val tracker: String?,
```

Comment:
```suggestion
    val tracker: String,
```
Трекер не может быть null